src/ExternalModule.ts

```diff
@@ -99,7 +99,12 @@ export default class ExternalModule {
 		});
 	}
 
-	traceExport(name: string): Variable {
+	traceExport(
+		name: string,
+		_options?: {
+			isExportAllSearch?: boolean;
+		}
```

Comment:
Could we make this a default flag over an object rather?

Reply:
I figured `module.traceExport(name, { isExportAllSearch: true });` would be more clear to the reader than, `module.traceExport(name, true);`, but I can change it if you want.

Reply:
That would be great just to avoid the object allocation, for what it's worth. It doesn't seem like the flags will be extending anytime soon.

Reply:
done